draft-ounsworth-csr-attestation.md

```diff
@@ -363,6 +363,61 @@ Columns:
 
 # Security Considerations
 
+A certification request consists of a distinguished name, a public key,
```

Comment:
#21 now uses "Certificate Signing Requests". Maybe:

```suggestion
A certificate signing request consists of a distinguished name, a public key,
```

Reply:
PKCS#10 defines a message and it is called "Certification Request". This message has later been called "Certificate Signing Request".

In this paragraph I am referring to PKCS#10 -- maybe I should include a reference.

Reply:
Proposal is here: https://github.com/lamps-wg/csr-attestation/pull/27